readthedocs/acl/utils.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib.auth import BACKEND_SESSION_KEY
+
+from readthedocs.acl.constants import BACKEND_REQUEST_KEY
+
+
+def get_auth_backend(request):
```

Comment:
I feel like we already have this code on .com -- this is porting it right?

Reply:
yeah, I'm porting this piece from .com, so we have all code here.